app/web/features/connections/friends/useFriendRequests.ts

```diff
@@ -41,10 +45,10 @@ export default function useFriendRequests(
           ...friendRequest,
           friend: usersData.get(friendRequest.userId),
         }))
-      : void 0;
+      : [];
 
   return {
-    data,
+    data: isLoading ? undefined : data,
```

Comment:
Wouldn't it be simpler to have the `undefined` (or `void 0`, we were going through a phase of using all the cool stuff we knew about JS lol, as in practice `undefined` will never be re-declared to warrant the need to use `void 0` and it only makes this more obscure perhaps 😅 ) on line 48 as before?

Reply:
Was wondering about the `void 0` haha. Okay so this made me find a lot of mistakes I made, but neither your comment or mine ended up being the answer 😅.

I had to adjust some behavior since I wanted `useLiteUsers` to return `undefined` if there's no data (instead of empty Map, a holdover from earlier). I also needed to return an empty array for `useFriendRequests` here if `useLiteUsers` returned `undefined` data.

I was incorrectly hacking it before I now realize, as I had something wrong in `userLiteUsers`.